<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node pkg="brain_control" type="brain_control_interface.py" name="brain_control_interface" output="screen">
  </node>

  <node pkg="brain_control" type="brain_control_cmd.py" name="brain_control_cmd" output="screen">
  </node>

</launch>